packages/quantic/force-app/main/default/lwc/quanticQueryError/quanticQueryError.js

```diff
@@ -0,0 +1,149 @@
+import { LightningElement, api, track } from 'lwc';
+import {registerComponentForInit, initializeWithHeadless} from 'c/quanticHeadlessLoader';
+import {I18nUtils} from 'c/quanticUtils';
+
+import coveoOnlineHelpLink from '@salesforce/label/c.quantic_CoveoOnlineHelpLink';
+import moreInformation from '@salesforce/label/c.quantic_MoreInformation';
+import checkForMore from '@salesforce/label/c.quantic_CheckForMore';
+import community from '@salesforce/label/c.quantic_Community';
+import contactCoveoSupportTeam from '@salesforce/label/c.quantic_ContactCoveoSupportTeam';
+import goBack from '@salesforce/label/c.quantic_GoBack';
+import disconnectedException from '@salesforce/label/c.quantic_DisconnectedException';
+import noEndpointsException from '@salesforce/label/c.quantic_NoEndpointsException';
+import invalidTokenException from '@salesforce/label/c.quantic_InvalidTokenException';
+import organizationIsPausedException from '@salesforce/label/c.quantic_OrganizationIsPausedException';
+import disconnectedTitle from '@salesforce/label/c.quantic_DisconnectedTitle';
+import noEndpointsTitle from '@salesforce/label/c.quantic_NoEndpointsTitle';
+import invalidTokenTitle from '@salesforce/label/c.quantic_InvalidTokenTitle';
+import organizationIsPausedTitle from '@salesforce/label/c.quantic_OrganizationIsPausedTitle';
+import genericErrorTitle from '@salesforce/label/c.quantic_GenericErrorTitle';
+import disconnectedDesc from '@salesforce/label/c.quantic_DisconnectedDesc';
+import noEndpointsDesc from '@salesforce/label/c.quantic_NoEndpointsDesc';
+import invalidTokenDesc from '@salesforce/label/c.quantic_InvalidTokenDesc';
+import organizationIsPausedDesc from '@salesforce/label/c.quantic_OrganizationIsPausedDesc';
+import genericErrorDesc from '@salesforce/label/c.quantic_GenericErrorDesc';
```

Comment:
I think after seeing this I would make an `errorUtils` component maybe? Just a js file that would import and export all the errors under 1 object maybe?

```
lwc
  └───quanticQueryError.js
    ├──errorLabels.js
```

```js
// errorLabels.js
import disconnectedException from '@salesforce/label/c.quantic_DisconnectedException';
import noEndpointsException from '@salesforce/label/c.quantic_NoEndpointsException';
import invalidTokenException from '@salesforce/label/c.quantic_InvalidTokenException';
[...]

export { 
  exceptions = {
    disconnectedException,
    ...
  },
  titles = { ... }
}

// quanticQueryError.js
import { exceptions, titles } from './errorLabels.js'

```

What do you think?